storage/src/vespa/storage/persistence/messages.cpp

```diff
@@ -177,4 +178,23 @@ AbortBucketOperationsCommand::makeReply() {
     return std::make_unique<AbortBucketOperationsReply>(*this);
 }
 
+std::unique_ptr<api::StorageReply>
+RunTaskCommand::makeReply() {
+    return std::make_unique<RunTaskReply>(*this);
+}
+
+RunTaskCommand::RunTaskCommand(const spi::Bucket &bucket, std::unique_ptr<spi::BucketTask> task)
+    : api::InternalCommand(ID),
+      _task(std::move(task)),
+      _bucket(bucket)
+{
+    assert(_task);
+}
+
+RunTaskCommand::~RunTaskCommand() = default;
+
+RunTaskReply::RunTaskReply(const RunTaskCommand& cmd)
+    : api::InternalReply(ID, cmd)
+{}
+
```

Comment:
Consider adding a simple override of `print` for both `RunTaskCommand` and `RunTaskReply` (can look at e.g. `GetIter(Command|Reply)::print` for inspiration). This will make it easier to identify these tasks when dumping persistence queues etc for debugging.

Otherwise we'll just see something like `InternalCommand(1011)`, which is less immediately obvious.

Reply:
Fixed